modules/sor/lib/pathGraph/pathGraph.ts

```diff
@@ -3,7 +3,7 @@ import { PathGraphEdgeData, PathGraphTraversalConfig } from './pathGraphTypes';
 import { BasePool } from '../poolsV2/basePool';
 import { PathLocal } from '../path';
 
-const DEFAULT_MAX_PATHS_PER_TOKEN_PAIR = 4;
+const DEFAULT_MAX_PATHS_PER_TOKEN_PAIR = 3;
```

Comment:
@franzns - moved it back to 3 because this fix might increase the number of paths significantly and I think it would be better to check the load on production before bumping to 4 again.
ps: 3 should be enough to cover the scenario we saw issues these past few days